main.js

```diff
@@ -576,10 +900,10 @@ function ScrollToTop() {
                 {dswContent.map(({ route, component, heading }, index) => {
                   return (
                     <Route key={`route-${index}`} path={route}>
-                      <div className="flex-col flex-shrink-0 pb-8" style={{fontFamily: "Overpass", color: "#152145"}} key={`section-${index}`}>
+                      <div className="flex-col flex-shrink-0 pb-8" style={{ fontFamily: "Overpass", color: "#152145" }} key={`section-${index}`}>
                         <h1 className="pb-4 uxw-text-display-2">{heading}</h1>
                         {component}
-                        <NextSectionLink index={index}/>
+                        <NextSectionLink index={index} />
```

Comment:
I think you can remove this line to remove the Next Section link

Reply:
done